src/main/java/racingcar/service/CarRacing.java

```diff
@@ -0,0 +1,66 @@
+package racingcar.service;
+
+import racingcar.model.Car;
+import java.util.*;
+
+public class CarRacing {
+
+    private List<Car> carsStatus;
+
+    public CarRacing() {}
+
+    public CarRacing(int numberOfCars) {
+        carRaceReady(numberOfCars);
+    }
+
+    private void carRaceReady(int numberOfCars) {
+        makeCarsStatusList(numberOfCars);
+    }
+
+    public void carRaceStart() {
+        int[] randomNumbers = generateRandomNumberForMovingCar(carsStatus.size());
+
+        boolean[] carsMovingForwardStatus = isCarMovingForward(randomNumbers);
+
+        for(int i = 0; i < carsMovingForwardStatus.length; i++) {
+            Car car = carsStatus.get(i);
+            boolean isCarMovingForward = carsMovingForwardStatus[i];
+            car.move(isCarMovingForward);
+        }
```

Comment:
```suggestion
        for (Car car : cars) { // enhanced for loop 으로 변경
            car.move(isCarMovingForward());
        }
```
carsStatus 대신 cars 표현이 더 일반적이지 않을까요? 
기존 변수 이름은 자동차의 어떤? 상태를 담고 있는 것으로 보여요

그리고 로직이 다소 복잡해 보입니다.
위와 같이 리팩토링해보면서 복잡도를 줄여볼 수 있을 것 같습니다.